assets/wizards/popups/utils.js

```diff
@@ -8,6 +8,17 @@ export const isOverlay = popup =>
 	[ 'top', 'bottom', 'center' ].indexOf( popup.options.placement ) >= 0;
 
 /**
+<<<<<<< HEAD
+ * Check whether the given popup is above-header.
+ *
+ * @param {object} popup Popup object to check.
+ * @return {boolean} True if the popup is a above-header, otherwise false.
+ */
+export const isAboveHeader = popup => 'above_header' === popup.options.placement;
+
+/**
+=======
+>>>>>>> master
```

Comment:
Some conflict resolution comment to delete ;)

Reply:
Whoops, weird how the linter didn't catch this. Fixed in eb09a1e.